server-main.go

```diff
@@ -61,6 +61,22 @@ ENVIRONMENT VARIABLES:
      MINIO_CACHE_SIZE: Set total cache size in NN[GB|MB|KB]. Defaults to 8GB.
      MINIO_CACHE_EXPIRY: Set cache expiration duration in NN[h|m|s]. Defaults to 72 hours.
 
+		 Example: export  MINIO_CACHE_EXPIRY=10hours
+		          export  MINIO_CACHE_SIZE=5GB
+
+  TRACING:
```

Comment:
I'm looking at open tracing and other better tracing tools. Will work on better tracing next
we can choose the approach that time may be